apps/patient/src/views/Pharmacy.tsx

```diff
@@ -299,13 +300,13 @@ export const Pharmacy = () => {
 
           // Send order placed sms to demo participant
           triggerDemoNotification(
-            '5416029101',
+            phone,
             'photon:order:placed',
             selectedPharmacy.name,
             formatAddress(selectedPharmacy.address)
           );
 
-          navigate(`/status?demo=true`);
+          navigate(`/status?demo=true&phone=${phone}`);
```

Comment:
why do we need phone number here?

Reply:
to trigger other sms, was thinking okay for now later i'll move to context